common/app/templates/inlineJS/blocking/membershipAccess.scala.js

```diff
@@ -29,13 +30,14 @@
 
             if (identity.isUserLoggedIn()) {
                 ajax({
-                    url: membershipUrl + '/user/me',
+                    url: membersDataApiUrl + '/user-attributes/me/membership',
                     type: 'json',
                     crossOrigin: true,
                     withCredentials: true
                 }).then(function (resp) {
+                    var access = resp.contentAccess
                     // Check the users access matches the content
-                    var canViewContent = (requiresPaidTier) ? !!resp.tier && resp.isPaidTier : !!resp.tier;
+                    var canViewContent = access && (requiresPaidTier) ? !!access.paidMember : !!access.member;
```

Comment:
... and in fact `access.paidMember` and `access.member` are both booleans anyway when returned from the API